content/influxdb/v2.0/tools/api/_index.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Working with the InfluxDB API
+seotitle: Use the InfluxDB API
+description: Interact with InfluxDB using a rich API for writing and querying data and more.
+weight: 3
+menu:
+  influxdb_2_0:
+    name: Working with the API
+    parent: Tools & integrations
+---
+
+InfluxDB offers a rich API that developers can use to create applications.
+The following pages offer general guides to the most commonly used API methods.
+
+For detailed documentation on the entire API, see [InfluxDBv2 API Documentation](/influxdb/v2.0/reference/api/#influxdb-v2-api-documentation)
```

Comment:
How about line 15 to bottom?

Reply:
At the moment, I think it would get lost under the query heading.